index.js

```diff
@@ -138,6 +138,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(require('hof-middleware-markdown')());
```

Comment:
There are two differences:

* In firearms that code is at the top level of the file (i.e. not inside a function call), so will execute exactly once at start-up and no more.
* In firearms the containing file is short, and the require is on line 33, and so is easily visible - especially given that most of the firearms file is require statements already.